contracts/system.go

```diff
@@ -0,0 +1,14 @@
+package contracts
```

Comment:
You don't need this file, this is going to be introduced with v3-contracts integration: https://github.com/0xPolygon/polygon-edge/pull/796/files#diff-cb0d9c0c8a217bb409554571a2a979efd58f1dbf13e059824db4b1df98abed3cR1